config/vufind/markdown.ini

```diff
@@ -30,3 +30,39 @@
 ; blocks are too deeply-nested.
 ;max_nesting_level = 10
 
+; Which extension you want to activate. List of extension names separated by comma. Available extensions as of league/commonmark
+; version 1.4: ExternalLink, HeadingPermalink, SmartPunct, TableOfContents
+; Some of them could have a configuration, see sections bellow
+;extensions = ExternalLink,HeadingPermalink,SmartPunct,TableOfContents
+
+; See https://commonmark.thephpleague.com/1.4/extensions/external-links/
+[ExternalLink]
+; This should be always set, if you wan't to use this extension. You can use regular
+; expressions to match group of hosts
+internal_hosts[] = www.example.com
+;open_in_new_window = true
+;html_class = external-link
+
+; See https://commonmark.thephpleague.com/1.4/extensions/heading-permalinks/
+[HeadingPermalink]
+;html_class = heading-permalink
+;id_prefix = user-content
+;inner_contents = '¶'
+;insert = before
+;title = Permalink
+
+; See https://commonmark.thephpleague.com/1.4/extensions/smart-punctuation/
+[SmartPunct]
+;double_quote_opener = '“'
+;double_quote_closer = '”'
+;single_quote_opener = '‘'
+;single_quote_closer = '’'
+
+; See https://commonmark.thephpleague.com/1.4/extensions/table-of-contents/
+[TableOfContents]
+;html_class = table-of-contents
+;position = top
+;style = bullet
+;min_heading_level = 1
+;max_heading_level = 6
+;normalize => relative
```

Comment:
It would also be good to have a newline on the end of this line for consistency with other files.

Reply:
Fixed